xbmc/games/ports/Port.cpp

```diff
@@ -105,6 +105,22 @@ bool CPort::OnAccelerometerMotion(const std::string& feature, float x, float y,
   return m_gameInput->OnAccelerometerMotion(feature, x, y, z);
 }
 
+bool CPort::OnWheelMotion(const std::string& feature, float position, unsigned int motionTimeMs)
+{
+  if ((position != 0.0f) && !m_gameClient.AcceptsInput())
```

Comment:
Any reason for the extra `()` around the first condition? (same below)

Reply:
I like to be explicit in order-of-operation situations.